README.md

```diff
@@ -53,5 +53,24 @@ open localhost:1313
 ### How to add new episode
 
 ```shell
-hugo new episodes/TITLE_OF_SHOW
+hugo new episodes/"TITLE OF SHOW.md"
 ```
+
+## Podcast release process
+
+### Preparations
+* Discuss the topic for the podcast in [Next Episode thread](https://devsecops.fm/episodes/next-episode/)
```

Comment:
if it has the same URL for each "next episode", will we keep the a single discussion on disqus for every "next episode" or not ?

Reply:
It depends on how Disqus id is being generated. In the old setup episode files where just digits so I could foresee what it will be and could pre-create the page for the episode ahead of time. Now we started with names as part of URL which makes it harder to predict the name of the episode. So if we keep everything as it is then we would have one page for all future episodes or we can go back to digits and then have a new next episode page for every episode or we can figure out something else. So we need to define what we want first. I'm fine with both options

Reply:
I don't really know how to deal with disqus in the scenario. More thinking is needed because I would like to have human readable URLs instead of numbers as it might be less error prone for sharing links.